src/libraries/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -474,6 +475,30 @@ public static void CopyResult_RawData()
             }
         }
 
+        [Fact]
+        public static void RawDataMemory_NoCopy()
+        {
+            using (X509Certificate2 cert = new X509Certificate2(TestData.MsCertificate))
+            {
+                ReadOnlyMemory<byte> first = cert.RawDataMemory;
+                ReadOnlyMemory<byte> second = cert.RawDataMemory;
+                Assert.True(first.Span == second.Span, "RawDataMemory returned different values.");
```

Comment:
It's quite rare to see someone use == with spans and have it actually be correct ;-)